.github/workflows/test-wdio.yml

```diff
@@ -0,0 +1,51 @@
+name: BrowserStack Tests
+
+on:
```

Comment:
Does this run ATM? I'm not seeing any logs from GH Actions

Reply:
The workflow is running and passing now: https://github.com/ionic-team/stencil/actions/runs/8252971259/job/22573834057?pr=5456